<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1391</width>
    <height>884</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>0</y>
      <width>1333</width>
      <height>897</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane
{
	border: 1px;
	background: rgb(35, 40, 49);
}
QTabBar::tab
{
	width: 330px;
	height:50px;
	background: rgb(76, 81, 93);
}
QTabBar::tab:selected
{
	background: rgb(35, 40, 49);
}
QTabBar::tab:hover
{
	background: rgb(56, 62, 73);
}</string>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <property name="iconSize">
     <size>
      <width>42</width>
      <height>43</height>
     </size>
    </property>
    <widget class="QWidget" name="grap">
     <attribute name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/free-icon-algebra-5090382.png</normaloff>:/images/free-icon-algebra-5090382.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Построить граффик</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="CalculatorTab">
     <attribute name="title">
      <string>Калькулятор</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="diag">
     <attribute name="title">
      <string>Диаграммы</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="setting">
     <attribute name="title">
      <string>О программе</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>431</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Актуальная версия программы: 0.01</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1391</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
